openstudio-building-energy-standard-measures.gemspec

```diff
@@ -39,7 +39,7 @@ Gem::Specification.new do |spec|
     spec.add_development_dependency 'rubocop', '~> 1.50'
 
     spec.add_dependency 'openstudio-extension', '~> 0.8.1'
-    spec.add_dependency 'openstudio-standards', '~> 0.6.3'
+    spec.add_dependency 'openstudio-standards', '~> 0.8.2'
```

Comment:
I wonder if this is necessary, or are there any way we can set dependency to above 0.6.3? `'> 0.6.3'` I guess this may do?